src/NServiceBus.Hosting.Tests/StartablesAndStoppables/StartableAndStoppableRunnerTimeoutTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace NServiceBus.Hosting.Tests
+{
+    using System;
+    using System.Threading.Tasks;
+    using Logging;
+    using Moq;
+    using NUnit.Framework;
+
+    public class StartableAndStoppableRunnerTimeoutTests
+    {
+        static TimeSpan TimeToWarn = TimeSpan.FromSeconds(1);
+        static TimeSpan TimeToWaitForTimeout = TimeToWarn.Add(TimeSpan.FromSeconds(2));
```

Comment:
I don't see a good reason why these need to be static fields and cannot be defined inside `RunSlowStartableTest`?